src/main/java/com/elisis/gtnhlanth/loader/RecipeLoader.java

```diff
@@ -987,11 +987,11 @@ public static void loadLanthanideRecipes() {
         // Digester to produce Lanthanum Chloride Concentrate
         GT_Values.RA.stdBuilder().itemInputs(GT_OreDictUnificator.get(OrePrefixes.crushed, Materials.Lanthanum, 1))
                 .itemOutputs(Materials.SiliconDioxide.getDust(3)).fluidInputs(Materials.Chlorine.getGas(36000))
-                .fluidOutputs(LanthanumChlorideConcentrate.getFluidOrGas(3000)).specialValue(800).eut(114514)
+                .fluidOutputs(LanthanumChlorideConcentrate.getFluidOrGas(3000)).specialValue(800).eut(122880)
                 .duration(40).addTo(RecipeAdder.instance.DigesterRecipes);
```

Comment:
decompose the tick duration in a unique h/min/s/t way, using the TICKS, SECONDS, MINUTES, HOURS constants, for instance 40 is 2*SECONDS